### PR TITLE
Wazo 2326 cdr with token in url

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

**Review by bloom1 — CHANGES_REQUESTED** [review]

File: wazo_call_logd/plugins/cdr/schemas.py [hunk 1]
```diff
@@ -99,6 +99,7 @@ class CDRListRequestSchema(Schema):
     recurse = fields.Boolean(missing=False)
     distinct = fields.String(validate=OneOf(['peer_exten']), missing=None)
     recorded = fields.Boolean(missing=None)
+    format = fields.String(validate=OneOf(['csv', 'json']), missing=None)
```
[the remark this answers is not in the document]
Comment (pc-m) [reply wazo_call_logd/plugins/cdr/schemas.py:102]: I added token, tenant and format to the changelog :+1:

**Comment by pc-m** [comment]
@bloom1 I fixed all reported issues and force pushed to look like a pro!

File: integration_tests/suite/test_cdr.py [hunk 2]
```diff
@@ -1447,3 +1448,255 @@ def test_list_multitenant(self):
...
+        **{'id': 12},
+        tenant_uuid=OTHER_TENANT,
+        date='2017-03-23 00:00:00',
+        participants=[{'user_uuid': OTHER_USER_UUID, 'line_id': '1', 'role': 'source'}],
+    )
+    @call_log(**{'id': 13}, date='2017-03-23 00:00:00', tenant_uuid=MASTER_TENANT)
+    def test_list_multitenant_token_and_tenant_as_query_string(self):
+        port = self.service_port(9298, 'call-logd')
+
+        response = requests.get(
```
Comment (bloom1) [inline integration_tests/suite/test_cdr.py:1474]: Why did you have to use requests here?

File: wazo_call_logd/plugins/cdr/http.py [hunk 3]
```diff
@@ -109,12 +109,33 @@ def _output_csv(data, code, http_headers=None):
     return response
 
 
-class CDRAuthResource(AuthResource):
-    representations = {'text/csv; charset=utf-8': _output_csv}
+def format_cdr_result(result):
+    is_csv = (
+        request.args.get('format') == 'csv'
+        or request.headers.get('Accept') == 'text/csv; charset=utf-8'
```
Comment (bloom1) [inline wazo_call_logd/plugins/cdr/http.py:115]: I'm really not sure if this is a good way to check for the accept header. Flask-restful, for example, uses `request.accept_mimetypes.best_match()`: https://github.com/flask-restful/flask-restful/blob/master/flask_restful/__init__.py#L494
What do you think about that?

File: integration_tests/suite/test_cdr.py (hunk 2 above)
Comment (pc-m) [reply integration_tests/suite/test_cdr.py:1474]: I wanted to be sure that the client would not add the `Wazo-Tenant` and the `X-Auth-Token` headers to the request.

File: wazo_call_logd/plugins/cdr/http.py (hunk 3 above)
Comment (pc-m) [reply wazo_call_logd/plugins/cdr/http.py]: Good point, I wondered if I was right when writing it and eventually forgot about it. Thanks for the link! :)

**Review by bloom1 — APPROVED** [review]

**Pull request merged**